lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp

```diff
@@ -203,6 +204,27 @@ struct CommandApdu
         return {0x00, 0xA4, p1, 0x0C, std::move(file)};
     }
 
+    static PCSC_CPP_CONSTEXPR_VECTOR CommandApdu selectEF(byte_type p1, byte_vector file)
```

Comment:
This should have a short comment that this is similar to `select()` above except `P2=0x04` returns FCI as well.